src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -388,25 +400,34 @@ export function getInputType(
             models.set(name, model);
 
             // Resolve properties after model is added to the map to resolve possible circular dependencies
-            addModelProperties(m.properties, properties, baseModel?.DiscriminatorPropertyName);
+            addModelProperties(
+                m.properties,
+                properties,
+                baseModel?.DiscriminatorPropertyName
+            );
 
             // Temporary part. Derived types may not be referenced directly by any operation
-            // We should be able to remove it when https://github.com/Azure/cadl-azure/issues/1733 is closed 
+            // We should be able to remove it when https://github.com/Azure/cadl-azure/issues/1733 is closed
             if (model.DiscriminatorPropertyName && m.derivedModels) {
-                m.derivedModels.forEach(dm => {
+                m.derivedModels.forEach((dm) => {
```

Comment:
Since you're changing this place, can we use `for .. of` here?

Reply:
updated.